### PR TITLE
ADR 60: Skin wearables

### PR DESCRIPTION
**Pull request by cazala**
ADR 60: Skin wearables

### CODE REVIEW

File: docs/ADR-60-skin-wearables-in-builder-editor.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Skin wearables in Builder editor
+
+## Context and Problem Statement
+
+We are adding a new `skin` category to the available wearable slots. This new slot will be used for wearables that shape the whole avatar. Here's a [link to the PRD](https://www.notion.so/decentraland/PRD-Skin-Wearable-Category-9913df0dd6944f839dc403310f151625) explaining the logic behind this new feature, but the scope of this ADR is to address the problem of how to support a new slot in the Builder editor.
```
Comment (nachomazzara) [inline docs/ADR-60-skin-wearables-in-builder-editor.md:5]: The notion link is private. We may need to remove it.

Comment (nachomazzara) [reply docs/ADR-60-skin-wearables-in-builder-editor.md:5]: This ADR should include support in the explorer too, not only the builder

**Review by nachomazzara — CHANGES_REQUESTED** [review]
Great!

File: docs/ADR-60-skin-wearables-in-builder-editor.md (hunk 1 above)
Comment (cazala) [reply docs/ADR-60-skin-wearables-in-builder-editor.md]: I added the changes required to the explorer, marketplace, nft-server and subgraphs as well.

**Review by nachomazzara — APPROVED** [review]
LGTM

**Pull request merged**